apps/organizations/utils.py

```diff
@@ -1,4 +1,13 @@
+# Django Imports
+from django.apps import apps
+from django.contrib.sites.shortcuts import get_current_site
+
 # HTK Imports
+from htk.apps.organizations.emailers import send_invitation_email
+from htk.apps.prelaunch.utils import (
+    PrelaunchSignup,
```

Comment:
I was previously experiencing a lot of issues with importing `PrelaunchSignup` model if the app is not actually installed. We should probably look into this, or import this at the function level and catch the import error.

Reply:
Done